packages/mc-scripts/src/utils/credentials-storage.spec.js

```diff
@@ -0,0 +1,94 @@
+const mock = require('mock-fs');
+const CredentialsStorage = require('./credentials-storage');
+const homedir = require('os').homedir();
+
+const fullCredentialsPath = `${homedir}/.commercetools/mc-credentials.json`;
+
+const getExpiryDate = (hours) => {
+  const date = new Date();
+  const expireTime = date.getTime() + 1000 * hours;
+  date.setTime(expireTime);
+  return date.toUTCString();
+};
+
+const credentialsStorage = new CredentialsStorage();
+
+afterEach(() => {
+  mock.restore();
+});
+
+const mcApiUrl = 'https://mc-api.europe-west1.gcp.commercetools.com';
+
+describe('When cookie is not expired', () => {
+  beforeEach(() => {
+    mock({
+      [fullCredentialsPath]: JSON.stringify({
+        [mcApiUrl]: {
+          sessionToken: 'hello-world',
+          expiresAt: getExpiryDate(36000),
+        },
+      }),
+    });
+  });
+
+  it('should return token', () => {
+    expect(credentialsStorage.getToken(mcApiUrl)).toBe('hello-world');
+  });
+
+  it('should return true', () => {
+    expect(credentialsStorage.isSessionValid(mcApiUrl)).toBe(true);
+  });
```

Comment:
Nit: I would combine these into one `it` test, as we're only doing assertions.

```suggestion
  it('should load credentials', () => {
    expect(credentialsStorage.getToken(mcApiUrl)).toBe('hello-world');
    expect(credentialsStorage.isSessionValid(mcApiUrl)).toBe(true);
  });
```